### PR TITLE
Added new git commands 

### PR DESCRIPTION
**Pull request by Nhoya**
This is related to #4 
Added 

`git init`
`git init --bare`
`git clone --bare`

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+- It will create a new git repository in the current directory.
+
+`$ git init --bare`
+- it will create a new bare repository (no .git folder, history is stored in the project root).
+
 `$ git clone <link to repo>`
 - Clones repository to local machine.
 
+`$ git clone --bare <link to reo>`
+- Do a bare copy of the repository you want to clone.
```
Comment (aSquare14) [inline main.md]: Write a line on how this is different from git clone !

Comment (Nhoya) [reply main.md]: Can I add a note at the end of the chapter?

File: main.md [hunk 2]
```diff
@@ -1,9 +1,18 @@
 ## GIT CHEATSHEET
 
 ### Common Commands
+`$ git init`
+- It will create a new git repository in the current directory.
```
Comment (aSquare14) [inline main.md]: Make a diferent section called "Initializing git" and put lines 4-9 there.

**Comment by vibhavagarwal5** [comment]
@Nhoya Since all your commits are similar...can u squash those?

File: main.md [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
+- It will create a new bare repository.
+
+A bare repository is a bit different from a regular one, it doesn't .git folder, history is stored in the project root, also if you try to `git clone --bare` from a remote repository (e.g. github) you will lost track of it's origin since usually bare repositories are supposed to be served at the users as remote endpoints.
+
+## Cloning Repositories
 `$ git clone <link to repo>`
 - Clones repository to local machine.
 
+### Shallow cloning
+$ git clone <link> --depth=1
```
Comment (vibhavagarwal5) [inline main.md]: fix this, add it in ``

Comment (Nhoya) [reply main.md]: > @Nhoya Since all your commits are similar...can u squash those?

All of them?

Comment (vibhavagarwal5) [reply main.md]: Chuck it..

Comment (Nhoya) [reply main.md]: Oh well, the rebase wasn't working as intended... probably ZSH is not working in my favour, let me just reopen a PR

**Comment by vibhavagarwal5** [comment]
fix conflicts

**Comment by Nhoya** [comment]
> fix conflicts

It should be ok now

**Comment by vibhavagarwal5** [comment]
Doesnt seem so..As u can still see it has conflicting files..

**Comment by aSquare14** [comment]
LGTM :tada:

File: main.md (hunk 1 above)
Comment (vibhavagarwal5) [reply main.md]: I think it would be better to add the difference here itself.

**Pull request merged**